File: src/main/java/problems/Uri1042_SortSimple.java
package problems;

import java.io.IOException;
import java.util.ArrayList;
import java.util.Collections;
import java.util.Comparator;
import java.util.List;
import java.util.Scanner;

public class Uri1042_SortSimple {
    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);

        int n1 = ler.nextInt();
        int n2 = ler.nextInt();
        int n3 = ler.nextInt();

        List<Integer> list = new ArrayList<Integer>();
        list.add(n1);
        list.add(n2);
        list.add(n3);

        sortAutoComplete(list);

        for (int i = 0; i < list.size(); i++) {
            System.out.println(list.get(i));
        }

        System.out.println();
        System.out.println(n1);
        System.out.println(n2);
        System.out.println(n3);
    }

    public static void sortAutoComplete(List<Integer> preferencia2) {
        Collections.sort(preferencia2, new Comparator<Integer>() {
            @Override
            public int compare(Integer auto2, Integer auto1) {
                return auto2 - auto1;
            }
        });
    }
}

File: src/main/kotlin/problems/Uri1042_SortSimple.kt
package problems

import java.util.Scanner

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)
    val list = listOf(ler.nextInt(), ler.nextInt(), ler.nextInt())

    for (i in list.sorted()) {
        println(i)
    }
    println()
    for (i in list) {
        println(i)
    }
}

/*
URI Online Judge | 1042
Sort Simples
Adaptado por Neilor Tonin, URI  Brasil

Timelimit: 1
Leia 3 valores inteiros e ordene-os em ordem crescente. No final, mostre os valores em ordem crescente, uma linha em branco e em seguida, os valores na sequência como foram lidos.

Entrada
A entrada contem três números inteiros.

Saída
Imprima a saída conforme foi especificado.

Exemplo de Entrada	Exemplo de Saída
7 21 -14

-14
7
21

7
21
-14

-14 21 7

-14
7
21

-14
21
7
 */